new-site/src/docs/elements/components/card/code.mdx

```diff
@@ -57,40 +64,52 @@ import Playground from '../../../../components/Playground';
   <div class="hds-card">
     <div class="hds-card__body">
       <div class="hds-card__heading" role="heading" aria-level="2">Card</div>
-      <div class="hds-card__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</div>
+      <div class="hds-card__text">
```

Comment:
The core rendering differs from React. Maybe some core styles are missing? Rebasing to master might help.

<img width="999" alt="image" src="https://user-images.githubusercontent.com/2777633/155751491-7d58f3d7-4adc-442f-ad60-2773171711df.png">

Reply:
- [ ] import core component styles